docs/dev/stream/operators/process_function.md

```diff
@@ -86,7 +86,7 @@ The following example maintains counts per key, and emits a key/count pair whene
     and emits the key/count if they match (i.e., no further update occurred during that minute)
 
 <span class="label label-info">Note</span> This simple example could have been implemented with
-session windows. We use `ProcessFunction` here to illustrate the basic pattern it provides.
+session windows. We use `KeyedProcessFunction` here to illustrate the basic pattern it provides.`KeyedProcessFunction`, as an extension of `ProcessFunction`.
```

Comment:
Actually, there is no any extension relationship for `KeyedProcessFunction` compared with `ProcessFunction`.  From [FLINK-8560](https://issues.apache.org/jira/browse/FLINK-8560) `KeyedProcessFunction` is more like an optimization.

Reply:
I agree with you. I'll change it right away